azure-functions/acbs-function/package.json

```diff
@@ -38,14 +38,14 @@
   "devDependencies": {
     "babel-jest": "29.5.0",
     "babel-loader": "9.1.2",
-    "eslint": "^8.57.0",
+    "eslint": "^8.57.1",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-import": "^2.30.0",
     "jest": "^29.7.0"
   },
   "engines": {
-    "node": ">=20",
-    "npm": ">=10"
+    "node": "22.8.0",
```

Comment:
is it worth doing >=22.8.0? I know a lot of the UKEF machines auto update sometimes and you don't always have control over the updates and rolling back so could be a pain to hard stick to a certain version